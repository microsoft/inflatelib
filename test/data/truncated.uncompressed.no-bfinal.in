
# This generates a file with a two uncompressed blocks, but neither has bfinal set to true
>1
0       # bfinal = false
00      # No compression

# NOTE: This will byte align, which is desired
>16
002B    # LEN = 43
FFD4    # NLEN = ~LEN

# And finally, the data
>8
"This is from the first uncompressed block.\n"

>1
0       # bfinal = false
00      # No compression

# NOTE: This will byte align, which is desired
>16
002C    # LEN = 44
FFD3    # NLEN = ~LEN

# And finally, the data
>8
"This is from the second uncompressed block.\n"
