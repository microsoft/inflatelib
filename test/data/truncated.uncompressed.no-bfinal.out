
"This is from the first uncompressed block.\n"
"This is from the second uncompressed block.\n"
